### PR TITLE
Refactor/ds 926 prefixes of utilities and helpers

### PR DESCRIPTION
**Pull request by literat**
### Issue reference

https://jira.lmc.cz/browse/DS-926

---

### Before submitting the PR, please make sure you do the following

- [ ] Read the [Contributing Guidelines](https://github.com/lmc-eu/spirit-design-system/blob/main/CONTRIBUTING.md).
- [ ] Follow the [PR Title/Commit Message Convention](https://github.com/lmc-eu/spirit-design-system/blob/main/CONTRIBUTING.md#commit-conventions).
- [ ] Provide a description in this PR that addresses **what** the PR is solving, or reference the issue that it solves (e.g. `fixes #123`).
- [ ] Ideally, include relevant tests that fail without this PR but pass with it.

### CODE REVIEW

**Comment by crishpeen** [comment]
Please see Breadcrumbs and FileUploaderInput too

File: packages/web-twig/src/Resources/components/FileUploader/FileUploaderAttachment.twig [hunk 1]
```diff
@@ -38,7 +39,7 @@
         <Icon name="{{ _iconName }}" isReusable={ false } />
     {% endif %}
     <span class="{{ _nameClassName }}">
-        <span class="text-truncate" data-spirit-populate-field="name">
+        <span class="{{ _textTruncate }}" data-spirit-populate-field="name">
             {{- _fileName -}}
         </span>
     </span>
```
Comment (crishpeen) [inline packages/web-twig/src/Resources/components/FileUploader/FileUploaderAttachment.twig:45]: Oh my, could you please add classes in the if _onEdit section to the {# Class names #} section too?

**Review by crishpeen — DISMISSED** [review]

File: packages/web-twig/src/Resources/components/FileUploader/FileUploaderAttachment.twig (hunk 1 above)
Comment (crishpeen) [reply packages/web-twig/src/Resources/components/FileUploader/FileUploaderAttachment.twig:45]: (it is below this code excerpt)

Comment (literat) [reply packages/web-twig/src/Resources/components/FileUploader/FileUploaderAttachment.twig:46]: Fixed, thanks for noticing :-)

**Review by adamkudrna — CHANGES_REQUESTED** [review]
There are still a few places to fix:

- Twig `FileUploaderInput` – link classes
- Twig `Breadcrumbs` – display classes
- Twig `FileUploader` – text-truncate

I hope the list is complete and correct 🙂

File: packages/web-twig/src/Resources/components/FileUploader/FileUploaderAttachment.twig [hunk 2]
```diff
@@ -12,7 +12,9 @@
 {%- set _rootClassName = _spiritClassPrefix ~ 'FileUploaderAttachment' -%}
 {%- set _imageClassName = _spiritClassPrefix ~ 'FileUploaderAttachment__image' -%}
 {%- set _nameClassName = _spiritClassPrefix ~ 'FileUploaderAttachment__name' -%}
-{%- set _removeClassName = _spiritClassPrefix ~ 'FileUploaderAttachment__action' -%}
+{%- set _actionClassName = _spiritClassPrefix ~ 'FileUploaderAttachment__action' -%}
+{%- set _slotClassName = _spiritClassPrefix ~ 'FileUploaderAttachment__slot' -%}
+{%- set _textTruncate = _spiritClassPrefix ~ 'text-truncate' -%}
```
[the remark this answers is not in the document]
Comment (literat) [reply packages/web-twig/src/Resources/components/FileUploader/FileUploaderAttachment.twig]: Fixed.

**Comment by literat** [comment]
@adamkudrna 
> There are still a few places to fix:
> 
> * Twig `FileUploaderInput` – link classes

Thanks. I did not notice. Fixed.

> * Twig `Breadcrumbs` – display classes

I think this should be a part of the new `BreadcrumbItem`, see https://jira.lmc.cz/browse/DS-835. Or do you think that we should fix this now?

> * Twig `FileUploader` – text-truncate

I am unable to find another occurrence of `text-truncate`. In `FileUploader.twig` there is no such class.

> I hope the list is complete and correct 🙂

**Comment by literat** [comment]
> I think this should be a part of the new `BreadcrumbItem`, see https://jira.lmc.cz/browse/DS-835. Or do you think that we should fix this now?

Well, I tried to do it right now. But it seems that it would be much bigger then I expected. So for now I have prefixed those classes and started to prepare the `BreadcrumbsItem` here https://github.com/lmc-eu/spirit-design-system/pull/1052.

File: packages/web-twig/src/Resources/components/Breadcrumbs/Breadcrumbs.twig [hunk 3]
```diff
@@ -22,12 +25,12 @@
...
             {% for item in _items %}
                 {% if loop.index is same as(_items|length - 1) and _goBackTitle is not same as('') %}
-                    <li class="d-tablet-none">
+                    <li class="{{ _displayTabletNoneClassName }}">
                         <Icon name="chevron-left" />
                         <Link href="{{ item.url }}" color="primary" isUnderlined>{{ _goBackTitle }}</Link>
                     </li>
                 {% endif %}
-                <li class="d-none d-tablet-flex">
+                <li class="{{ [_displayNoneClassName, _displayTabletFlexClassName] | join(' ') }}">
```
Comment (adamkudrna) [inline packages/web-twig/src/Resources/components/Breadcrumbs/Breadcrumbs.twig:33]: I think our `classProp()` function can be applied here, but it doesn't matter if the component overhaul is already in progress.

**Review by adamkudrna — APPROVED** [review]
Cool!

File: packages/web-twig/src/Resources/components/FileUploader/FileUploaderInput.twig [hunk 4]
```diff
@@ -78,7 +80,7 @@
     <div class="{{ _dropzoneClassName }}" data-spirit-element="dropZone">
         <Icon name="{{ _iconName }}" isReusable={false} />
         <label for="{{ _id }}" class="{{ _dropzoneLabelClassName }}">
-            <span class="{{ _linkClassName }} link-primary link-underlined">
+            <span class="{{ [_linkClassName, _linkPrimaryClassName, _linkUnderlinedClassName] | join(' ') }}">
```
[the remark this answers is not in the document]
Comment (literat) [reply packages/web-twig/src/Resources/components/FileUploader/FileUploaderInput.twig]: Yep, but will be refactored soon.

File: packages/web-twig/src/Resources/components/Breadcrumbs/Breadcrumbs.twig (hunk 3 above)
Comment (literat) [reply packages/web-twig/src/Resources/components/Breadcrumbs/Breadcrumbs.twig]: Yep, but will be refactored soon.

**Pull request merged**